Latest Material2 Types Incorrectly Implement 'CanDisable'

#### Bug, feature request, or proposal:
Since upgrading material beta from 2.0.0-beta.3 to 2.0.0-beta.4, I'm getting multiple errors for button.d.ts, checkbox.d.ts, radio.d.ts, slide-toggle.d.ts, and slider.d.ts when building.

#### What is the expected behavior?
The application should build fine with no errors. 

#### What is the current behavior?
When building, I'm getting errors similar to this:
```
/node_modules/@angular/material/typings/button/button.d.ts(40,22): error TS2420: Class 'MdButton' incorrectly implements interface 'CanDisable'.
  Property 'disabled' is missing in type 'MdButton'.

/node_modules/@angular/material/typings/button/button.d.ts(40,39): error TS2507: Type '(new (...args: any[]) => CanDisable) & typeof MdButtonBase' is not a constructor function type.
```
These errors are the same for all 5 components mentioned at the top of this issue.
#### Which versions of Angular, Material, OS, browsers are affected?
Angular 4.0.2, Material 2.0.0-beta.4, Windows 7 & 10, Chrome

#### Is there anything else we should know?
I have made sure to remove the deprecated "forRoot()", and I do not use HttpModule from MdIconModule, which are the only breaking changes listed in the changelog.

--- a/.github/ISSUE_TEMPLATE.md
+++ b/.github/ISSUE_TEMPLATE.md
@@ -16,7 +16,7 @@ Plunker template: https://goo.gl/DlHd6U
 #### What is the use-case or motivation for changing an existing behavior?
 
 
-#### Which versions of Angular, Material, OS, browsers are affected?
+#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
 
 
 #### Is there anything else we should know?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,8 @@
 ### Breaking Changes
 
 * `MdIconModule` no longer imports `HttpModule`. If your application depended on `Http` being provided through `MdIconModule`, you should now directly import `HttpModule` into your application.
-* The `forRoot` method on all Angular Material modules has been removed. It was previously deprecated and a no-op. Importing the modules directly will have the same effect. 
+* The `forRoot` method on all Angular Material modules has been removed. It was previously deprecated and a no-op. Importing the modules directly will have the same effect.
+* Angular Material now requires TypeScript 2.2, which adds support for mixins. 
 
 ### Highlights
 

